src/search.cpp

```diff
@@ -62,10 +62,22 @@ namespace {
 
   // Different node types, used as a template parameter
   enum NodeType { NonPV, PV };
+  enum MarginType {Razor, FutilityChild, FutilityParent, Null, SeeRoot, SeeNonRoot, MarginsNB = 6};
```

Comment:
Right, if eventually it is taken into consideration I will follow your suggestion.